src/features/congregation/upcoming_events/upcoming_event_date/useUpcomingEventDate.tsx

```
@@ -0,0 +1,41 @@
+  allLocales = locales;
+});
+
+const useUpcomingEventDate = (date: Date) => {
+  const { t } = useAppTranslation();
+  const weekdays = generateWeekday();
+
+  //TODO: Update and remove allLocales
+  const eventDate = useMemo(() => {
+    const shortMonth = formatDate(date, 'LLL', {
+      locale:
+        allLocales && allLocales[t('tr_iso')]
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)